packages/core/database.old/lib/connector-registry.js

```diff
@@ -49,7 +49,7 @@ const createConnectorRegistry = ({ defaultConnection, connections }) => {
       return _connectors.get(defaultConnector);
     },
 
-    getByConnection(connection) {
+    getByConnection(connection = 'default') {
```

Comment:
this code isn't used anymore lookis like you don't need to change anything in database.old